chapter15/dtj/ex1

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use 5.010001;
+
+my $guess;
+my $secret =  int(1 + rand 100);
+
+while (1) {
+  say "Take a guess";
+  chomp ($guess = <STDIN>);
+
+  given ($guess) {
+    when ( $guess < $secret )   { say "Too low"; }
+    when ( $guess > $secret )   { say "Too High"; }
+    when ( $guess == $secret )  { say "Correct!"; last; }
+    default                     { say "Quitting!"; last; }
```

Comment:
I can confirm that I totally forgot to test that you can quit it, the idea was if it was empty/not a number then it would quit, but it probably doesn't quite work like that.